src/bin/add/main.rs

```diff
@@ -154,12 +149,8 @@ fn main() {
     if let Err(err) = handle_add(&args) {
         eprintln!("Command failed due to unhandled error: {}\n", err);
 
-        for e in err.iter().skip(1) {
-            eprintln!("Caused by: {}", e);
-        }
-
-        if let Some(backtrace) = err.backtrace() {
-            eprintln!("Backtrace: {:?}", backtrace);
```

Comment:
is it not possible to keep backtraces with anyhow? :(

Reply:
anyhow can display backtraces as long as it was complied by nightly, since the stdlib `backtrace` feature is still unstable. I've tried to convince `failure::Fail::backtrace` to accept an `anyhow::Error` and try to get a backtrace out of it, but haven't had any luck. `anyhow::Error` will deref to a `dyn Error + Send + Sync + 'static`, but `Fail::backtrace` complains that it is unsized.

I've added a feature that will allow a user to compile with backtrace support as long as they are on nightly, but if there is a better way I'd be more than happy to use it.

Reply:
@killercup are backtraces used for identifying bugs in cargo-edit if it fails to run on some input? maybe it's not that helpful for the users after all

Reply:
I could have sworn that anyhow brought in a dependency on `backtrace-rs` on stable but apparently not. I mean to shamelessly plug my own stuff but if you want stable backtraces you can swap from `anyhow` to `eyre` which is a fork that adds support for custom context types, which you could define to include a `backtrace::Backtrace`

Reply:
Losing backtraces is kind of a shame from a technical standpoint, but I don't think they are being relied on by anyone using cargo-edit, because we have such amazing error descriptions! :wink:

Kidding aside, I think this is fine. When we finally end up packaging binaries we can compile them with nightly and get some backtraces for free.

@yaahc, I just got what the name is about. Amazing. I'm also totally fine with using your fork. Or switching to it in a later PR.